### PR TITLE
feat(param-model): DEVX-164: Enable model detail view

### PR DESCRIPTION
**Pull request by catfishbluefish**
This enabled the model detail view in the two column layout as per DEVX-164 customer request

screen shot 2018-06-18 at 4 56 45 pm

### CODE REVIEW

File: src/kongComponents/model-example.jsx [hunk 1]
```diff
@@ -33,11 +33,31 @@ export default class ModelExample extends React.Component {
...
+          <a className="tablinks" data-name="example" onClick={ this.activeTab }>Example Value</a>
+        </li>
+        { schema ? <li className={ "tabitem" + ( !isExecute && this.state.activeTab === "model" ? " active" : "") }>
+          <a className={ "tablinks" + ( isExecute ? " inactive" : "" )} data-name="model" onClick={ this.activeTab }>Model</a>
+        </li> : null }
+      </ul>
       <div>
-        {example}
+        {
+          (isExecute || this.state.activeTab === "example") && example
```
Comment (nateslo) [inline src/kongComponents/model-example.jsx]: My React is super rusty, but will this render `false` if  ` (isExecute || this.state.activeTab === "example") ` resolves to false?

File: src/kongComponents/model-example.jsx [hunk 2]
```diff
@@ -23,8 +23,8 @@ export default class ModelExample extends React.Component {
     }
   }
 
-  activeTab = (e) => {
-    let { target: { dataset: { name } } } = e
+  activeTab =( e ) => {
+    let { target : { dataset : { name } } } = e
```
Comment (nateslo) [inline src/kongComponents/model-example.jsx]: This is some funky styling on the colons, is this a common style in this repo?

Comment (catfishbluefish) [reply src/kongComponents/model-example.jsx]: This is default code from swagger

**Comment by catfishbluefish** [comment]
Updated styles

New Preview:
screen shot 2018-06-22 at 11 06 52 am

File: src/kongComponents/model-example.jsx (hunk 2 above)
Comment (nijikokun) [reply src/kongComponents/model-example.jsx]: horrifying

**Pull request merged**